available_brick_points/app.py

```diff
@@ -0,0 +1,98 @@
+__author__ = "Anand Krishnan Prakash"
+__email__ = "akprakash@lbl.gov"
+
+import pymortar
+import datetime
+import pandas as pd
+import argparse
+
+def get_query():
+    query = """SELECT ?point ?point_type WHERE { ?point rdf:type/rdfs:subClassOf* brick:Point . ?point rdf:type ?point_type . };"""
+    return query
+
+def get_all_points(client, site=None):
+    query = get_query()
+
+    resp = client.qualify([query])
```

Comment:
Can you explain what this logic is doing? I think it might be easier to just supply `[site]` to the Fetch request, rather than going through qualify. If you want to run the app on all sites that have data/points, then you should use the Qualify call.

Reply:
You're right. But the Qualify call is useful when the user does not specify a particular site.